docs/io/visualization/sdec_plot.ipynb

```diff
@@ -13,8 +13,8 @@
    "execution_count": null,
```

Comment:
<p>Why not directly assign vertical slices of <code>data</code>  to both <code>observed_spectrum_wavelength</code>  and <code>observed_spectrum_flux</code>. Please look into ways of indexing ndarray to learn how to do it. If that's not possible with numpy, there's always pandas to do read almost all file formats. then you can just assign series to both variables.</p><p> </p><p>Always remember, iterating is the last we should do with numpy &amp; pandas as it reduces speed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1761/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>